apps/web/src/views/Swap/components/AccessRisk/index.tsx

```diff
@@ -7,19 +7,20 @@ import {
   Flex,
   HelpIcon,
   IconButton,
-  Link,
   RefreshIcon,
   Tag,
   Text,
   useTooltip,
   promotedGradient,
+  AutoRenewIcon,
 } from '@pancakeswap/uikit'
-import { useMemo, useState } from 'react'
+import { useEffect, useMemo, useState } from 'react'
 import { useUserTokenRisk } from 'state/user/hooks/useUserTokenRisk'
 import { useAllLists } from 'state/lists/hooks'
 import styled from 'styled-components'
 import useSWRImmutable from 'swr/immutable'
-import { fetchRiskToken, TOKEN_RISK } from 'views/Swap/hooks/fetchTokenRisk'
+import { fetchRiskToken } from 'views/Swap/hooks/fetchTokenRisk'
```

Comment:
Is this hook related to swap feature? If not can we treat it as a common hook and move it to `src/hooks`?